src/UriTemplate.php

```diff
@@ -122,23 +131,25 @@ private static function expandMatch(array $matches, array $variables): string
                 continue;
             }
 
+            /** @var mixed */
```

Comment:
I think you are missing the `$variable` here.

Reply:
No, when you omit the variable name, you refer to what is below, just like when doccing class properties.